ql/src/java/org/apache/hadoop/hive/ql/exec/vector/VectorizationContext.java

```diff
@@ -4611,7 +4611,14 @@ public String toString() {
     }
     sb.append("sorted projectionColumnMap ").append(sortedColumnMap).append(", ");
 
+    if (initialColumnNames == null) {
+      throw new IllegalStateException("initialColumnNames have not been initialized.");
```

Comment:
Does the configuration parameter directly relate to these 2 variables? If so can you also point out which configuration parameter in the message?

Reply:
It seems this issue does not have any directly related configuration parameters. So I didn't mention the parameter in the message.